app/assets/javascripts/wizard/components/wizard-step.js

```diff
@@ -26,6 +26,7 @@ const alreadyWarned = {};
 export default Component.extend({
   classNames: ["wizard-step"],
   saving: null,
+  stylingDropdown: {},
```

Comment:
You should initialize objects (and arrays) in an `init` method otherwise each instance of the component will end up with a reference to the same thing. I think in this case it'll be fine, but it'll help in the future if someone decides to mutate that object.